3_transformer.go

```diff
@@ -133,6 +139,15 @@ func Transform(g *client.GitClient, splices []*ExtractSplice) ([]*TemplateContex
 				if remoteURL, err = url.Parse(fmt.Sprintf("%s://%s%s", remoteURL.Scheme, remoteURL.Host, urlPath)); err != nil {
 					return nil, errors.WithStack(err)
 				}
+			case "file":
```

Comment:
Can you provide an example URL for testing? I'm curious how it parses into the file protocol

Reply:
Our remotes look like: `org-1232456@github.com:USER/REPO.git` and the url parser utility doesn't really know how to deal with them: https://go.dev/play/p/7jUtvntFvdl